tpl/config-wizard.html.twig

```diff
@@ -196,6 +196,10 @@ auth_opt_cacheseconds 300</textarea></p>
 				<p>{{ __('- Refresh hash and symlinks to your certificates', 'flyvemdm') }}</p>
 				<p>c_rehash /etc/mosquitto/certs</p>
 				<p>{{ __('Use a certificate signed by a certified authority or you may have trouble with the android devices, using them with custom certification authorities might not work (not tested).', 'flyvemdm') }}</p>
+				<blockquote><p>{{ __('After sending the certification request, the CA will very likely send back several files.') }}</p>
+				<p>{{ __('One is the certificate, signed by the CA, and the others are intermediate certificates.') }}</p>
+				<p>{{ __('In Mosquitto the certificate must be the concatenation of the certificate delivered + the intermediate certificates.') }}</p>
+				<p>{{ __('The operating system must contain more certificates to establish a trust chain to the root certificate.') }}</p></blockquote>
```

Comment:
Add an extra word here, for clarification

```The **client** operating system must contain more certificates to establish a trust chain``` (because the client operating system is either Android, either Linux for the backend.)